### ISSUE TITLE
Landscape infolist space issue

### ISSUE THREAD

**Issue by ShibajiCh** [jurialmunkey/skin.arctic.horizon.2#80]
Hi @jurialmunkey,

I noticed that in landscape infolist view if the number of seasons or episodes is less then there is extra empty space above the content instead of below. Could you please check? See screenshot below.
![IMG20220606201849](https://user-images.githubusercontent.com/65297828/172185509-9d15a7ee-282c-44c1-87dc-06df70ae61c5.jpg)

**Comment by jurialmunkey** [comment]
Yeah it's to do with how Kodi renders fixed position lists. Not much can be done about it.

The only "fix" is to change from a centered fixed position list to a standard list - but that means that then you lose the centered scrolling.

**Comment by ShibajiCh** [comment]
Ok. Fine. Thanks for the explanation!
